CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+If you want to create a PR and check your code locally you can use the following commands 
+to approcimate the checks the [CI](.github/workflows/ci.yml) workflow will perform. 
```

Comment:
`approcimate` 

Typo?

Reply:
Resolved in https://github.com/tandemdrive/syslog-fmt/pull/3/commits/f186113be41d31f7924a088307ec1932768b3122